### PR TITLE
Add EndlessLearningCollection view

### PR DESCRIPTION
**Pull request by danigm**
https://phabricator.endlessm.com/T33342

### CODE REVIEW

File: kolibri_explore_plugin/views.py [hunk 1]
```diff
@@ -92,3 +98,53 @@ def get(self, request, app):
...
+        job_ids = []
+
+        for c in channels:
+            task = {
+                "channel_id": c["id"],
+                "channel_name": c["name"],
+                "baseurl": self.BASE_URL,
+                "started_by_username": "endless",
+                "type": "REMOTECHANNELIMPORT",
+            }
```
Comment (dylanmccall) [inline kolibri_explore_plugin/views.py:135]: Let's replace the single-letter variable name here :)

```suggestion
        for channel in channels:
            task = {
                "channel_id": channel["id"],
                "channel_name": channel["name"],
                "baseurl": self.BASE_URL,
                "started_by_username": "endless",
                "type": "REMOTECHANNELIMPORT",
            }
```

**Review by dylanmccall — CHANGES_REQUESTED** [review]
Alas, I didn't spot any easy fix for that issue you noted with the setChannelInfo action. I just have some pedantic formatting details for you :)

**Comment by danigm** [comment]
Thanks for the suggestions! I've applied all of them. I'll try to fix the reloading issue during this week, and if "it's not possible" to do it in an easy way, we can always fallback to a full page reload on finish.

File: kolibri_explore_plugin/assets/src/components/InstallContentModal.vue [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+      },
+      buttonStyles() {
+        const stop = this.progress;
+        const color2 = 'rgba(255, 255, 255, 0.2)';
+        const color1 = 'rgba(255, 255, 255, 0)';
+        return {
+          background:
+            'linear-gradient(90deg,' +
+            `${color1} 0%, ${color1} ${stop}%,` +
+            `${color2} ${stop}%, ${color2} 100%)`,
```
Comment (manuq) [inline kolibri_explore_plugin/assets/src/components/InstallContentModal.vue:127]: there are better ways to do this than hardcoding the CSS, but it's totally fine for an MVP

File: kolibri_explore_plugin/assets/src/components/InstallContentModal.vue [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  }
+
+  .fade-enter-active,
+  .fade-leave-active {
+    @include transition($transition-fade);
+  }
+  .fade-enter,
+  .fade-leave-to {
+    opacity: 0;
+  }
```
Comment (manuq) [inline kolibri_explore_plugin/assets/src/components/InstallContentModal.vue:228]: Why are all these definitions needed for transitions? They happen automatically in the About modal.

**Review by manuq — APPROVED** [review]
This looks good code-wise. I want to discuss about releasing this in the ticket.

File: kolibri_explore_plugin/assets/src/components/InstallContentModal.vue (hunk 3 above)
Comment (danigm) [reply kolibri_explore_plugin/assets/src/components/InstallContentModal.vue]: I've removed the `fade` css classes to use the default.

**Pull request merged**